buildSrc/src/main/groovy/org/apache/beam/gradle/Repositories.groovy

```diff
@@ -54,6 +54,9 @@ class Repositories {
         url "https://packages.confluent.io/maven/"
         content { includeGroup "io.confluent" }
       }
+      maven {
+        url 'https://linkedin.jfrog.io/artifactory/flink-li-custom/'
```

Comment:
nit: should we also have a name for the repository?

Reply:
I thought of adding but then realized we are not referring to the repository in other parts of the build script. Also, other maven repos in the file did not have a <name>.So, did not add it.